src/server/scripts/Northrend/VioletHold/boss_xevozz.cpp

```diff
@@ -225,72 +243,132 @@ class npc_ethereal_sphere : public CreatureScript
     {
         npc_ethereal_sphereAI(Creature* creature) : ScriptedAI(creature)
         {
-            Initialize();
```

Comment:
by removing the call to Initialize() you left all the class fields uninitialized, triggering static analysis issues